.pre-commit-config.yaml

```diff
@@ -4,11 +4,18 @@ repos:
   rev: 4.3.21
   hooks:
   - id: isort
+    # FIXME: isort can be enabled for cython once v5.0.0 is out
+    # files: >-
+    #   ^.*\.p(xd|y|yx)$
+    # types: [file]
 
 - repo: git://github.com/asottile/add-trailing-comma
   rev: v0.7.1
   hooks:
   - id: add-trailing-comma
+    files: >-
+      ^.*\.p(xd|y|yx)$
+    types: [file]
```

Comment:
this doesn't work fwiw, add-trailing-comma uses the ast to find the things to fix

Reply:
But also this doesn't fail...

Reply:
because syntax errors are currently silently ignored because people got grumpy when it was trying to fix invalid code 🤷